vm/src/cformat.rs

```diff
@@ -420,25 +435,22 @@ impl CFormatString {
             mut tuple_index: usize,
         ) -> PyResult<usize> {
             match q {
-                Some(CFormatQuantity::FromValuesTuple) => {
-                    match elements.next() {
-                        Some(width_obj) => {
-                            tuple_index += 1;
-                            if !objtype::isinstance(&width_obj, &vm.ctx.types.int_type) {
-                                Err(vm.new_type_error("* wants int".to_owned()))
-                            } else {
-                                // TODO: handle errors when truncating BigInt to usize
-                                *q = Some(CFormatQuantity::Amount(
-                                    objint::get_value(&width_obj).to_usize().unwrap(),
-                                ));
-                                Ok(tuple_index)
-                            }
+                Some(CFormatQuantity::FromValuesTuple) => match elements.next() {
+                    Some(width_obj) => {
+                        tuple_index += 1;
+                        if !objtype::isinstance(&width_obj, &vm.ctx.types.int_type) {
```

Comment:
swapping if and else would make else block a lot cleaner.
but it seems it is not part of this PR